ossapi/models.py

```diff
@@ -378,12 +378,16 @@ class Search:
 class Spotlight:
     end_date: Datetime
     id: int
-    mode_specific: int
+    mode_specific: bool
     participant_count: Optional[int]
-    name: int
+    name: str
     start_date: Datetime
     type: str
 
+@dataclass
+class Spotlights:
+    spotlights: List[Spotlight]
+
```

Comment:
to be honest I'd probably do away with this model entirely and just instantiate `List[Spotlight]` - I know that's not what the osu-web docs say but doesnt really make much sense, they return lists of models in other places

Reply:
Is the goal to maintain 1:1 consistency with the docs or what's most logical? I agree the `Spotlights` model doesn't really make much sense

Reply:
consistency would be ideal but at the end of the day we're an api wrapper being consumed by other applications which don't care about the details of the api. Part of our job is to do what makes the most sense for consumers, even if it means we have to do more work to make it match the api.